js/data.js

```diff
@@ -110,27 +117,67 @@ function viewSwap(dataView) {
   if (dataView === 'profile') {
     $editProfile.className = 'hidden';
     $profile.className = '';
+    $entries.className = 'hidden';
+    $createEntry.className = 'hidden';
+    $entriesLink.className = '';
     data.view = 'profile';
     $profile.removeChild($profile.childNodes[0]);
     $profile.appendChild(renderProfile());
   }
+  if (dataView === 'entries') {
+    $editProfile.className = 'hidden';
+    $profile.className = 'hidden';
+    $entries.className = '';
+    $createEntry.className = 'hidden';
+    data.view = 'entries';
+  }
+  if (dataView === 'create-entry') {
+    $editProfile.className = 'hidden';
+    $profile.className = 'hidden';
+    $entries.className = 'hidden';
+    $createEntry.className = '';
+    data.view = 'create-entry';
+  }
```

Comment:
So I can approve this code, however, I want you to attempt to do this in a loop, imagine you had 100 views, wouldn't this be redundant?
You can try to do it in a loop, where you would hide every view, and then unhide the one the user is supposed to see. 

Feel free to give it a shot, you're approved either way